src/components/v5/common/ActionSidebar/partials/forms/AdvancedPaymentForm/hooks.ts

```diff
@@ -1,33 +1,34 @@
 import { useCallback, useMemo } from 'react';
 import { Id } from '@colony/colony-js';
 import { DeepPartial } from 'utility-types';
+import { useWatch } from 'react-hook-form';
 import { ActionTypes } from '~redux';
 import { mapPayload, pipe } from '~utils/actions';
-import {
-  useColonyContext,
-  useEnabledExtensions,
-  useNetworkInverseFee,
-} from '~hooks';
+import { useColonyContext, useNetworkInverseFee } from '~hooks';
 import { getCreatePaymentDialogPayload } from '~common/Dialogs/CreatePaymentDialog/helpers';
 import { ActionFormBaseProps } from '../../../types';
-import { useActionFormBaseHook } from '../../../hooks';
-import { DECISION_METHOD_OPTIONS } from '../../consts';
+import {
+  DecisionMethod,
+  DECISION_METHOD,
+  useActionFormBaseHook,
+} from '../../../hooks';
 import { AdvancedPaymentFormValues, validationSchema } from './consts';
+import { DECISION_METHOD_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
```

Comment:
This import should be grouped above with the rest of its kind :)

Reply:
Fixed